src/main/java/frc/robot/commands/AutoScoreAndBalance.java

```diff
@@ -14,9 +14,12 @@ public class AutoScoreAndBalance extends SequentialCommandGroup {
   private PathPlannerTrajectory traj =
       PathPlanner.loadPath(
           "ScoreAndBalanceTraj",
-          new PathConstraints(Cal.PLACEHOLDER_DOUBLE, Cal.PLACEHOLDER_DOUBLE));
+          new PathConstraints(
+              Cal.SwerveSubsystem.MAX_LINEAR_SPEED_METERS_PER_SEC,
+              Cal.SwerveSubsystem.MAX_LINEAR_ACCELERATION_METERS_PER_SEC_SQ));
 
   public AutoScoreAndBalance(boolean isFirstPath, Lift lift, DriveSubsystem drive) {
+    // TODO update this since scoreGamePiece and doneScoring are stubbed
```

Comment:
VS Code doesn't underline `// TODO` in blue, only `//TODO`. Idk if that matters, though.

Reply:
IDK I've been keeping track of our TODOs just by searching them, we do still have a number of them to burn down